### PR TITLE
Add appstream data

### PR DESCRIPTION
**Pull request by fgaz**
Needed for #535 

I used https://www.freedesktop.org/software/appstream/metainfocreator/ to generate it.

For reference, it also provided some meson code to validate the file and generate translations:

```meson
# Validate MetaInfo file
metainfo_file = '/path/to/app.organicmaps.desktop.metainfo.xml'
ascli_exe = find_program('appstreamcli', required: false)
if ascli_exe.found()
  test('validate metainfo file',
        ascli_exe,
        args: ['validate',
               '--no-net',
               '--pedantic',
               metainfo_file]
  )
endif
```

> AppStream MetaInfo files are recognized by xgettext, so you just need to add app.organicmaps.desktop to your POTFILES.in to generate templates for all translatable elements in the file. When building the software, you can then instruct your build system to merge in the translations into the new file before installing the translated file as /usr/share/metainfo/app.organicmaps.desktop.metainfo.xml.

```meson
# Localize a MetaInfo file and install it
i18n = import('i18n')

# NOTE: Remember to add the XML file to POTFILES.in!
metainfo_file = '/path/to/app.organicmaps.desktop.metainfo.xml'
i18n.merge_file(
    input:  metainfo_file,
    output: 'app.organicmaps.desktop.metainfo.xml',
    type: 'xml',
    po_dir: join_paths (meson.source_root(), 'po'),
    install: true,
    install_dir: join_paths (get_option ('datadir'), 'metainfo')
)
```

The file should be installed as `/usr/share/metainfo/app.organicmaps.desktop.metainfo.xml`. Could you point me to where I should add the cmake install directive?

Note that I set the metadata license to cc0, please tell me if I have to change it

cc @biodranik

### CODE REVIEW

File: qt/res/app.organicmaps.OrganicMaps.metainfo.xml [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+    <control>keyboard</control>
+    <control>touch</control>
+  </supports>
+  
+  <description>
+    <p>
+      # Organic Maps
+    </p>
+    <p>
+      [Organic Maps](https://organicmaps.app) is a free Android &amp; iOS offline maps app for travelers, tourists, hikers, and cyclists. It uses crowd-sourced OpenStreetMap data and is developed with love by <em>
```
Comment (andrewshadura) [inline qt/res/app.organicmaps.OrganicMaps.metainfo.xml:22]: I’m not sure you really want to include Markdown in HTML

Comment (fgaz) [reply qt/res/app.organicmaps.OrganicMaps.metainfo.xml]: Yeah, it looks like that generator did a pretty bad job. I went over the file and fixed that and other problems

File: qt/res/app.organicmaps.OrganicMaps.metainfo.xml [hunk 2]
```diff
@@ -0,0 +1,107 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+  <id>app.organicmaps.OrganicMaps</id>
```
Comment (biodranik) [inline qt/res/app.organicmaps.OrganicMaps.metainfo.xml:3]: I would use app.organicmaps.desktop

Comment (fgaz) [reply qt/res/app.organicmaps.OrganicMaps.metainfo.xml]: Done

**Review by biodranik — APPROVED** [review]

**Pull request merged**